examples/db-templates/mongodb-template.json

```diff
@@ -1,127 +1,150 @@
 {
-  "apiVersion": "v1beta1",
   "kind": "Template",
+  "apiVersion": "v1beta3",
   "metadata": {
+    "name": "mongodb",
+    "creationTimestamp": null,
```

Comment:
Since these are our examples, can we clean them in such a way that empty values (null, "", {}, etc.) are not here? IMHO that should make those samples more readable, it should contain just the necessary bits needed to create an origin object. Maybe this should be included in that conversion tool of yours?